lib/oban/job.ex

```diff
@@ -643,4 +642,18 @@ defmodule Oban.Job do
         {:error, "unknown option, #{inspect(option)}"}
     end)
   end
+
+  defp validate_keys(keys, unique) do
```

Comment:
Moved this to a separate function after CI complaining about cyclomatic complexity.